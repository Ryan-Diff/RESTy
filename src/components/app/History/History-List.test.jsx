
import React from 'react';
import { render, cleanup } from '@testing-library/react';
import HistoryList from './History-List';

describe('HistoryList component', () => {
  afterEach(() => cleanup());
  it('render HistoryList component', () => {
    const { asFragment } = render(<HistoryList
    histories={[{
        url:"https://placekitten.com/",
        method:"GET"
    }]}    
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});