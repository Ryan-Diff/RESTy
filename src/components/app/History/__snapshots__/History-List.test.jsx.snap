// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryList component render HistoryList component 1`] = `
<DocumentFragment>
  <ul>
    <li>
      History
    </li>
    <li>
      <h2>
        GET
      </h2>
      <p>
        https://placekitten.com/
      </p>
    </li>
  </ul>
</DocumentFragment>
`;
